omnibus/config/projects/agent.rb

```diff
@@ -115,22 +115,25 @@
 # creates required build directories
 dependency 'datadog-agent-prepare'
 
-# Datadog agent
-dependency 'datadog-agent'
-
 # Additional software
 dependency 'datadog-pip'
-dependency 'datadog-agent-integrations'
 dependency 'jmxfetch'
 
 # External agents
 dependency 'datadog-trace-agent'
 dependency 'datadog-process-agent' # Includes network-tracer
 
+# Integrations-core
+dependency 'datadog-agent-integrations'
+
 if osx?
   dependency 'datadog-agent-mac-app'
```

Comment:
nit: it'd be nice to keep this one below `datadog-agent` because it depends on `datadog-agent` (omnibus will re-order the definitions anyway, but it's still nice for readability to keep things close to the actual build order)

Reply:
You are right, done!